src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{color =' " + color + " ', number= " + number + "}";
+    }
+}
```

Comment:
Each file must have one blank line at the end. When you enter the github code, there should not be such a red icon, as well as empty lines after our code. One empty line is enough for this.